apps/liquidity-provision-dashboard/src/app/components/indicator/indicator.tsx

```diff
@@ -0,0 +1,24 @@
+import type { MarketTradingMode } from '@vegaprotocol/types';
+
+import { getColorForStatus } from '../../lib/utils';
+
+export const Indicator = ({
```

Comment:
I'm not using the Indicator from ui-toolkit because it has margin top hardcoded and there are no way to pass in other colors or classnames

Reply:
suggestion (non blocking): Our primitive components shouldn't deal  with spacing if possible. We should refactor the indicator to not have the margins instead the spacing should be applied by the parent